docs/new-architecture-library-intro.md

```diff
@@ -48,11 +50,29 @@ export interface Spec extends TurboModule {
 export default (TurboModuleRegistry.get<Spec>('<MODULE_NAME>'): ?Spec);
 ```
 
+##### Turbo Module: TypeScript Example
```

Comment:
I think we might want to use a tabbed snippet, like we do for Kotlin/Java. See here:
- https://github.com/facebook/react-native-website/pull/2977